src/jou_compiler.h

```diff
@@ -106,6 +108,7 @@ struct Constant {
         struct { int width_in_bits; bool is_signed; long long value; } integer;
         char *str;
         char double_text[100];  // convenient because LLVM wants a string anyway
+        char float_text[100]; // same as double
```

Comment:
Why do we need two of these? Maybe just have one `float_or_double_text` member?

Reply:
Merge double_text and float_text? 
Seems like a good idea.